From 02c77258793755eca5e68ca4bb4b5ee2e815c10b Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <erik.faye-lund@collabora.com>
Date: Tue, 23 Jan 2024 19:55:15 +0100
Subject: [PATCH] mesa/main: fix regression in extension-checking

This condition accidentally got inverted when cleaning up code, whoops.

Fixes: 3251f321b86 ("mesa: some cleanups for texparam extension checks")
Reviewed-by: Alyssa Rosenzweig <alyssa@rosenzweig.io>
---
 src/mesa/main/texparam.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mesa/main/texparam.c b/src/mesa/main/texparam.c
index dcb03f8cac9e1..747bc29355c49 100644
--- a/src/mesa/main/texparam.c
+++ b/src/mesa/main/texparam.c
@@ -2762,7 +2762,7 @@ get_tex_parameteriv(struct gl_context *ctx,
          break;
 
       case GL_TEXTURE_CUBE_MAP_SEAMLESS:
-         if (_mesa_has_AMD_seamless_cubemap_per_texture(ctx))
+         if (!_mesa_has_AMD_seamless_cubemap_per_texture(ctx))
             goto invalid_pname;
          *params = (GLint) obj->Sampler.Attrib.CubeMapSeamless;
          break;
-- 
GitLab

